File: javascript/SelectionSort.js
function SelectionSort(arr) {
  let temp;
  let minimum;
  for (let i = 0; i < arr.length; i++) {
    minimum = i;
    for (let j = i + 1; j < arr.length; j++) {
      if (arr[minimum] > arr[j]) {
        minimum = j;
      }
    }
    if (i != minimum) {
      temp = arr[i];
      arr[i] = arr[minimum];
      arr[minimum] = temp;
    }
  }

  console.log(`Sorted array:- ${arr}`);
}

SelectionSort([345, 51, 1, 99, 7, 3, 5, 7, 46, 34, 76, 22, 78, 34, 78, 98]);

File: CSharp/Ds_and_algo/SelectionSort.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Ds_and_algo
{
    class SelectionSort
    {
        public int[] arr;
        public SelectionSort(int[] _arr)
        {
            arr = _arr;
        }

        public void Sort()
        {
            int minimum;
            int temp;
            for (int i = 0; i < arr.Length; i++)
            {
                minimum = i;
                for (int j = i + 1; j < arr.Length; j++)
                {
                    if (arr[minimum] > arr[j])
                    {
                        minimum = j;
                    }
                }

                if (i != minimum)
                {
                    temp = arr[i];
                    arr[i] = arr[minimum];
                    arr[minimum] = temp;
                }

            }
            Console.WriteLine($"Sorted array:- {string.Join(", ", arr)}");

        }
    }
}
